### PR TITLE
Add Contributions by CroutonDigital

### PR DESCRIPTION
**Pull request by romanv1812**
Contributions by CroutonDigital for delegation program

### CODE REVIEW

**Review by jcstein — APPROVED** [review]

**Comment by jcstein** [comment]
hi @romanv1812 i tried to resolve conflicts, but ran into issue with edit access:

```
Switched to branch 'main'
Your branch is up to date with 'origin/main'.
Already up to date.
ERROR: Permission to Crouton-Digital/awesome-celestia.git denied to jcstein.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

can you please resolve conflicts? thanks!

**Comment by romanv1812** [comment]
Hello @jcstein, conflicts resolved.

**Comment by CroutonDigital** [comment]
> hi @romanv1812 i tried to resolve conflicts, but ran into issue with edit access:
> 
> ```
> Switched to branch 'main'
> Your branch is up to date with 'origin/main'.
> Already up to date.
> ERROR: Permission to Crouton-Digital/awesome-celestia.git denied to jcstein.
> fatal: Could not read from remote repository.
> 
> Please make sure you have the correct access rights
> and the repository exists.
> ```
> 
> can you please resolve conflicts? thanks!

@jcstein  I resolve access issue, can you try again? 

thanks!

**Comment by jcstein** [comment]
every time a new PR is merged, conflicts will arise and I will still not have write access to resolve. can you please modify this PR so I have write access to modify it? thanks! then I'll resolve issues.

**Comment by CroutonDigital** [comment]
@jcstein Thank you! 

I fixed and merge PR. Can you check? 

Thank you!

File: README.md [hunk 1]
```diff
@@ -959,4 +961,6 @@ Prebuilt Binary: https://storage.crouton.digital/testnet/celestia/bin/celestia-a
 #### Community contributions
 Helped to prepare this video about Celestia:   
 https://www.youtube.com/watch?v=nmpfNXf797U
+=======
```
Comment (jcstein) [inline README.md:964]:
```suggestion
```

this is from resolving conflicts

**Review by jcstein — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -114,29 +114,6 @@ An Awesome List of Celestia Resources
 
 ## Node operator contributions
 
-### Contributions by [Stakin](https://stakin.com)
```
Comment (jcstein) [inline README.md:117]: why's this removed?

**Comment by jcstein** [comment]
it will be easiest to open a PR and just put your contributions at the end of the list, with the same format as others. thanks!

**Comment by maximdogonov** [comment]
I'm sorry, let try again fix my PR. 
Thank you!

**Comment by jcstein** [comment]
no worries @maximdogonov !

File: README.md [hunk 3]
```diff
@@ -949,34 +950,63 @@ Speaker French Event WEB3connect :
 
 ---
 
-### List of Contributions from [Encapsulate (fka KingSuper)](https://encapsulate.xyz/)
```
Comment (jcstein) [inline README.md:952]: now it is removing kingsuper. please just reopen a new pr like https://github.com/celestiaorg/awesome-celestia/pull/65

**Review by jcstein — CHANGES_REQUESTED** [review]

**Pull request closed without merging**